vpdq/cpp/CMakeLists.txt

```diff
@@ -63,18 +63,28 @@ add_executable(vpdq-hash-video
     bin/vpdq-hash-video.cpp
 )
 
+
 target_link_libraries(vpdqlib
     PkgConfig::LIBAV
 )
 
-target_link_libraries(vpdq-hash-video
+target_link_libraries(match-hashes-byline
     vpdqlib
+    PkgConfig::LIBAV
 )
 
 target_link_libraries(match-hashes-brute
     vpdqlib
+    PkgConfig::LIBAV
 )
 
-target_link_libraries(match-hashes-byline
+target_link_libraries(vpdq-hash-video
     vpdqlib
+    PkgConfig::LIBAV
 )
+
+# Pass the include and libav* library directories to setup.py for Python binding
+set_target_properties(vpdqlib
+    PROPERTIES
+    ENVIRONMENT "LIBRARY_DIRS=$<TARGET_PROPERTY:PkgConfig::LIBAV,IMPORTED_LOCATION>;INCLUDE_DIRS=$<TARGET_PROPERTY:PkgConfig::LIBAV,INTERFACE_INCLUDE_DIRECTORIES>"
```

Comment:
`LIBRARY_DIRS` and `INCLUDE_DIRS` are passed as env variables to `setup.py`

This is how setup.py gets the libav* library locations.